doc/os_requirements.md

```diff
@@ -59,6 +59,15 @@ feature enabled.
 cargo build --features systemd
 ```
 
+To allow systemd to receive status changes from service, you need to add `NotifyAccess=exec` option 
```

Comment:
```suggestion
To allow systemd to receive status changes from service, you need to set the `NotifyAccess` option to either `exec` or `main`. 
```

Reply:
I believe `main` should be sufficient. Since we are the only process that should be started by the service.

Reply:
We can leave both options in the doc, so the users can make a choice according to their needs.